components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -131,6 +131,17 @@ const ContributorHighlightCard = ({
     return !matches ? false : true;
   };
 
+  const getEmojiReactors = (id: string) => {
```

Comment:
```suggestion
    const getEmojiReactors = (reaction_users: string[]) => {
    if (reaction_users.length > 3) {
      return `${reaction_users.slice(0, 3).join(", ")} and ${reaction_users.length - 3} others`;
    } else if (reaction_users.length > 1) {
      return ``;
    } else if (reaction_users.length === 1) {
      return `${reaction_users[0]}`;
    }
  };
```

Reply:
I want to suggest this change since this particular function is being where we already have the data, we don't need to search for it again. You can also make use of `reaction_count` in this function but since it is a string, you need to convert it though.
It also increases readability of the function. Which led to me find the next bug.
This way, we can also bring this function outside of a react component as well which is always good I think.

Reply:
There's one important bug related to this function. It is not handling the case for when there are three reactions on a highlight.

![image](https://github.com/open-sauced/app/assets/27003616/e2e6d204-055a-4004-8f91-3f9f211019a4)